flagsmith/flagsmith.py

```diff
@@ -237,7 +228,9 @@ def get_environment_flags(self) -> Flags:
     def get_identity_flags(
         self,
         identifier: str,
-        traits: typing.Optional[typing.Mapping[str, TraitValue]] = None,
+        traits: typing.Optional[UserTraits] = None,
```

Comment:
```suggestion
        traits: typing.Optional[IdentityTraits] = None,
```

We should avoid the use of `User` here - it's a conscious decision to use `Identity` instead of `User` everywhere, because an identity isn't always a user.

Reply:
I meant an SDK user, as in "user provided traits". I'll change to something less confusing.

Reply:
Done.